test/e2e/webdriver/driver.js

```diff
@@ -390,9 +390,23 @@ class Driver {
     return elements.map((element) => wrapElementWithAPI(element, this));
   }
 
-  async clickElement(rawLocator) {
-    const element = await this.findClickableElement(rawLocator);
-    await element.click();
+  async clickElement(rawLocator, retries = 3) {
+    for (let attempt = 0; attempt < retries; attempt++) {
+      try {
+        const element = await this.findClickableElement(rawLocator);
+        await element.click();
+        return;
+      } catch (error) {
+        if (
+          error.name === 'StaleElementReferenceError' &&
+          attempt < retries - 1
+        ) {
+          await this.driver.delay(1000);
```

Comment:
Was this what was intended?
```
await this.delay(1000);
```

I'm not seeing a `delay` function on the internal webdriver instance. I'm seeing a `this.driver.delay is not a function` error on various e2e jobs

Reply:
oops that is a great catch! indeed it should be `this.delay()`

Reply:
here is the fix
https://github.com/MetaMask/metamask-extension/pull/24838

thank you v much Mark :pray:

Reply:
on the bright side, this means that this race-condition was happening quite often 😅